src/js/views/channel_slider_view.js

```diff
@@ -61,6 +61,58 @@ var ChannelSliderView = Backbone.View.extend({
         }
     },
 
+    hexToRgb: function hexToRgb(hex) {
+        // handle #ff00ff
+        var result = /^#?([a-f\d]{2})([a-f\d]{2})([a-f\d]{2})$/i.exec(hex);
+        if (result) return {
+            r: parseInt(result[1], 16),
+            g: parseInt(result[2], 16),
+            b: parseInt(result[3], 16)
+        };
+        // handle #ccc
+        result = /^#?([a-f\d]{1})([a-f\d]{1})([a-f\d]{1})$/i.exec(hex);
+        if (result) return {
+            r: parseInt(result[1]+'0', 16),
+            g: parseInt(result[2]+'0', 16),
+            b: parseInt(result[3]+'0', 16)
+        };
+    },
+
+    isDark: function(color) {
+        if (color.endsWith('.lut')) {
+            return false;
+        }
+        var c = this.hexToRgb(color);
+        var min, max, delta;
+        var v, s, h;
+        min = Math.min(c.r, c.g, c.b);
+        max = Math.max(c.r, c.g, c.b);
+        v = max;
+        delta = max-min;
```

Comment:
if min = max, then delta will be 0, leading to error below

Reply:
It won't actually lead to an error since a ```number/0``` returns ```Infinity``` in JavaScript:
```
>>> h = 2 / 0
Infinity
>>> h = h * 60
Infinity
>>> h = h/360
Infinity
>>> h > 0.6
true
```

Reply:
still a division by 0 is not correct. I will not rely on the interpretation of JS, this could change    and maybe lead to error

Reply:
Fixed in fcf8fa4